lib/mix/lib/mix/cli.ex

```diff
@@ -126,4 +133,13 @@ defmodule Mix.CLI do
       ["--version", "-v"], do: :version
 
   defp check_for_shortcuts(_), do: nil
+
+  defp usage() do
```

Comment:
We can change `mix --help` to show the usage but `mix help help` is definitely not useful as it is a bit meta: it is the help for the help task.